src/components/InfoFooter.vue

```diff
@@ -66,6 +66,19 @@ export default {
       defaultRelease: 'https://github.com/sanger/traction-ui/releases',
     }
   },
+  computed:{
+        footerClasses(){
+            let bgColorClass = '';
+            if (this.environment == 'development'){
+                bgColorClass = 'bg-blue-600';
+            }else if(this.environment == 'staging'){
+                bgColorClass = 'bg-purple-600';
+            }else if(this.environment == 'production'){
+                bgColorClass = 'bg-gradient-to-tr';
+            }
+            return [ 'border border-t-2 pt-5 pb-3 flex-shrink-0 border-sdb-100', bgColorClass ];
+        }
```

Comment:
Just a thought - since we are accessing this across multiple places, perhaps we can move this to a common place using composables.